draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -162,19 +162,19 @@ in an Informational Internet Draft in 2015, published as an
 Informational RFC in 2017 {{?RFC8312}} and then as a Proposed
 Standard in 2023 {{?RFC9438}}.
 
-BBR is developed as an internal research project by Google,
+BBR was developed as an internal research project by Google,
```

Comment:
I think "is developed" is actually correct. There is ongoing active development, entirely at Google, and periodic reports to the IRTF/ICCRG or IETF/CCWG. Change control is firmly by Google, and there have been no attempt to make this an IETF project.

Reply:
Hmmm... somebody needs to decide on the form of words. Would this word: " At the time this document was published, BBR is being developed as"....?

Reply:
At the time of writing, ... has been applied, see latest PR.